Validation/MajorRoads/UBA-DE_HVS_to_END_DF7_10_Roads.halex.alignment.xml

```diff
@@ -48,30 +33,80 @@ withTransformationContext {&#13;
 	&#13;
 	_target {&#13;
 		actionPlanId_identifier( c.Allgemeines.kennungPlan.value() )&#13;
-		competentAuthorityIdIdentifier(  c.Allgemeines.kennungBehoerde.value() )&#13;
+		limitValues_otherCriteriaDescription( c.Allgemeines.infoGrenzwert.value() )&#13;
+		limitValues_noiseLimitReportIdIdentifier( 'LR_DE_00_1' )&#13;
 	}&#13;
-}&#13;
-
+}
 </core:text>
         </complexParameter>
     </cell>
-    <cell relation="eu.esdihumboldt.cst.functions.groovy.create" id="Cb31e8faf-97e1-47c6-b625-dd9c31c191d1" priority="normal">
+    <cell relation="eu.esdihumboldt.cst.functions.groovy.create" id="Cd2c0eb42-1857-4e92-ac44-3c6b1d6ab98f" priority="normal">
         <target>
             <class>
-                <type name="NAP_MajorRoadLimitValues"/>
+                <type name="NAP_MajorRoad_LAU"/>
             </class>
         </target>
         <complexParameter name="script">
             <core:text xmlns:core="http://www.esdi-humboldt.eu/hale/core" xml:space="preserve">
 withTransformationContext {&#13;
 	def c = _.context.collector(it)&#13;
 	&#13;
-	_target {&#13;
-		actionPlanId_identifier( c.Allgemeines.kennungPlan.value() )&#13;
-		limitValues_otherCriteriaDescription( c.Allgemeines.infoGrenzwert.value() )&#13;
-		limitValues_noiseLimitReportIdIdentifier( 'LR_DE_00_1' )&#13;
+	def ags_rs = c.Allgemeines.ags_rs.value()&#13;
+		&#13;
+	switch (ags_rs.length()) {&#13;
+		case 8:		&#13;
+			// AGS of Gemeinde is equivalent to LAU code	&#13;
+			_target {&#13;
+				actionPlanId_identifier( c.Allgemeines.kennungPlan.value() )&#13;
+				allInLAUCode( 'yes' )&#13;
```

Comment:
The value here should be `Yes` (see `CODELIST_Boolean()` helper)

Reply:
Good catch, thanks for the hint. I corrected it now, also for the two other cases